csv/src/main/java/org/apache/metamodel/csv/CsvConfiguration.java

```diff
@@ -80,13 +83,15 @@ public CsvConfiguration(int columnNameLineNumber, String encoding, char separato
     
     public CsvConfiguration(int columnNameLineNumber, String encoding, char separatorChar, char quoteChar,
             char escapeChar, boolean failOnInconsistentRowLength, boolean multilineValues) {
-        this(columnNameLineNumber, null, encoding, separatorChar, quoteChar, escapeChar, failOnInconsistentRowLength,
-                multilineValues);
+        this(columnNameLineNumber, null, null, encoding, separatorChar, quoteChar, escapeChar,
+                failOnInconsistentRowLength, multilineValues);
     }
 
-    public CsvConfiguration(int columnNameLineNumber, ColumnNamingStrategy columnNamingStrategy, String encoding,
-            char separatorChar, char quoteChar, char escapeChar, boolean failOnInconsistentRowLength,
-            boolean multilineValues) {
+
+    public CsvConfiguration(int columnNameLineNumber, ColumnNamingStrategy columnNamingStrategy,
```

Comment:
By replacing the old public constructor with this constructor, compile problems can occur in other projects using this library. Can you please retain the old constructor and have it invoke this constructor with a `null` value for the `ColumnTypeStrategy` parameter?